imagevector/images.yaml

```diff
@@ -128,7 +114,7 @@ images:
   sourceRepository: github.com/kubernetes-csi/external-provisioner
   repository: registry.k8s.io/sig-storage/csi-provisioner
   tag: "v3.6.1"
-  targetVersion: ">= 1.24"
+  targetVersion: ">= 1.25"
```

Comment:
I think we can remove this line altogether. There is no other csi-provisioner ref in the images.

Reply:
Done.